src/nodejs/server.ts

```diff
@@ -87,6 +90,12 @@ export class NstrumentaServer {
       options.allowCrossProjectApiKey !== undefined ? options.allowCrossProjectApiKey : false;
     this.allowUnverifiedConnection =
       options.allowUnverifiedConnection !== undefined ? options.allowUnverifiedConnection : false;
+    this.status = {
```

Comment:
was it necessary to change status to a class member in this PR?  seems to make sense to do so, but is not obvious to me why it is in this PR